webportal/src/app/cluster-view/services/services.component.js

```diff
@@ -35,24 +36,13 @@ const serviceViewHtml = serviceViewComponent({
 
 const loadServices = () => {
   loading.showLoading();
-  $.ajax({
-    //url: `${webportalConfig.restServerUri}/api/v1/job`,
-    //type: 'GET',
-    success: (data) => {
-      loading.hideLoading();
-      if (data.error) {
-        alert(data.message);
-      } else {
-        $('#service-table').html(serviceTableComponent({
-          services: data      
-        }));
-      }
-    },
-    error: (xhr, textStatus, error) => {
-      const res = JSON.parse(xhr.responseText);
-      alert(res.message);
-    }
+  service.getServiceView('${webportalConfig.k8sApiServerUri}', 'default', (data) => {
```

Comment:
you need to uncomment `require webportal.config.json` line first to use `webportalConfig`.
`'${webportalConfig.k8sApiServerUri}'` doesn't work, use variable `webportalConfig.k8sApiServerUri` straightly.

Reply:
Resolved. Thanks